Project_source/app/src/main/java/android/clicker/school_live_simulator/GuitarCourseState.kt

```diff
@@ -0,0 +1,26 @@
+package android.clicker.school_live_simulator
+
+abstract class GuitarCourseState: StudyCourseState() {
+    abstract override val price: Int
```

Comment:
do we need override here?

Reply:
I think that it is not necessary. 
@nikitochka-akaka can you remove overriding this field from this abstract class?